.azure-pipelines/release-test.yml

```
@@ -0,0 +1,188 @@
+pr: none
+trigger: none
+
+strategy:
+  matrix:
+    Windows PS 5.1.14:
+      Image_Name: 'vs2017-win2016'
+      PSVersion: '5.1.14'
+  
+    Windows PS 6.2.4:
+      Image_Name: 'vs2017-win2016'
+      PSVersion: '6.2.4'
+     
+    Windows PS 7.0.3:
+      Image_Name: 'vs2017-win2016'
+      PSVersion: '7.0.3'
+
+    Linux PS 6.2.4:
+      Image_Name: 'ubuntu-18.04'
+      PSVersion: '6.2.4'
+      
+    Linux PS 7.0.3:
+      Image_Name: 'ubuntu-18.04'
+      PSVersion: '7.0.3'
+  
+    MacOS PS 6.2.4:
+      Image_Name: 'macOS-10.15'
+      PSVersion: '6.2.4'
+
+    MacOS PS 7.0.3:
+      Image_Name: 'macOS-10.15'
+      PSVersion: '7.0.3'
+
+pool:
+  vmImage: $(Image_Name)
+
+steps:
+- task: PowerShell@2
+  displayName: Prepare Powershell $(PSVersion) 
+  inputs:
+    filePath: 'tools/Test/SmokeTest/PrepareRequiredPowershell.ps1'
+    arguments: '-RequiredPsVersion $(PSVersion)'      
+      
+- task: DownloadPipelineArtifact@2
+  condition: and(succeeded(), eq(variables['GalleryName'], 'LocalRepo'))
+  displayName: Download Artifacts from Build Pipeline
+  inputs:
+    buildType: specific
+    project: $(ProjectToDownloadArtifacts)
+    definition: $(BuildPipelineName)
+    buildVersionToDownload: latestFromBranch
+    branchName: $(BranchName)
+    artifactName: $(ArtifactName)
+    targetPath: $(Pipeline.Workspace)
+
+- task: PowerShell@2
+  condition: and(succeeded(), eq(variables['GalleryName'], 'LocalRepo'))
+  displayName: Copy Artifacts to LocalRepo
+  inputs:
+    targetType: 'inline'
+    script: |
+     New-Item -Path "$(Pipeline.Workspace)" -Name "LocalRepo" -ItemType "directory"
+     Copy-Item -Path "$(Pipeline.Workspace)\\*.nupkg" -Destination "$(Pipeline.Workspace)\\LocalRepo\\"
+     Get-ChildItem "$(Pipeline.Workspace)\\LocalRepo\\" -Depth 2
+
+# - task: AzureCLI@2
+#   condition: and(succeeded(), eq(variables['GalleryName'], 'LocalRepo'))
+#   displayName: 'Prepare local repo'
+#   inputs:
+#     azureSubscription: '$(StorageSubscription)'
+#     scriptType: bash
+#     scriptLocation: inlineScript
+#     inlineScript: |
+#      cd "$(Pipeline.Workspace)"
+#      az storage copy -s "$(LocalRepoStorageLink)" -d "./LocalRepo"
+
+- task: NuGetCommand@2
+  condition: and(succeeded(), eq(variables['GalleryName'], 'LocalRepo'))
+  displayName: 'Download Previous Release .nupkg Files'
+  inputs:
+    command: custom
+    arguments: 'install Az -directdownload -packagesavemode nupkg -source https://www.powershellgallery.com/api/v2 -OutputDirectory packages'
+
+- task: PowerShell@2
+  condition: and(succeeded(), eq(variables['GalleryName'], 'LocalRepo'))
+  displayName: 'Copy Previous Release .nupkg files to LocalRepo'
+  inputs:
+    targetType: 'inline'
+    script: |
+      Get-ChildItem $(System.DefaultWorkingDirectory)/packages -Recurse -Include *.nupkg
```

```suggestion
      Get-ChildItem $(System.DefaultWorkingDirectory)/packages -Recurse -Include *.nupkg | Sort-Object -Property Name  | Format-Table -Property Name,Directory
```
Suggest a cleaner output